src/ui/issuepanel/IssuePanel.java

```diff
@@ -122,13 +135,30 @@ private void setupListView() {
 		setupKeyboardShortcuts();
 		listView.setOnItemSelected(i -> {
 			TurboIssue issue = listView.getItems().get(i);
-			ui.triggerEvent(new IssueSelectedEvent(issue.getId(), columnIndex));
+			nextIssueId = issue.getId();
+			nextColumnIndex = columnIndex;
+			timer.restart();
+			if (timer.isPaused()) {
+				timer.resume();
+			}
 			if (issueHasNewComments(issue)) {
 				issueCommentCounts.put(issue.getId(), issue.getCommentCount());
 				refreshItems();
 			}
 		});
 	}
+
+	private TickingTimer createTickingTimer() {
+		return new TickingTimer("Browser Request Delay Timer", BROWSER_REQUEST_DELAY, integer -> {
+            // do nothing for each tick
+        }, () -> {
+			if (nextIssueId >= 0) {
+				System.out.println("navigating to issue #" + nextIssueId + " from column: " + nextColumnIndex);
+				ui.triggerEvent(new IssueSelectedEvent(nextIssueId, nextColumnIndex));
```

Comment:
Anything that transitively modifies JavaFX UI elements needs to be run on the UI thread. There's a nasty race condition here because the timer's `onTimeout` runs in its own thread, and the issue-selection event eventually leads to UI operations.

Also, stray print statement.